model_draft_transactions.go

```diff
@@ -293,12 +305,37 @@ func (m *DraftTransaction) createTransactionHex(ctx context.Context) (err error)
 	return
 }
 
+func (m *DraftTransaction) addIncludeUtxos(ctx context.Context) error {
+	// Whatever utxos are selected, the IncludeUtxos should be added to the transaction
+	// This can be used to add for instance tokens where fees need to be paid from other utxos
+	// The satoshis of these inputs are not added to the reserved satoshis. If these inputs contain satoshis
+	// that will be added to the total inputs and handled with the change addresses.
+	includeUtxos := make([]*Utxo, 0)
+	for _, utxo := range m.Configuration.IncludeUtxos {
+		utxoModel, err := getUtxo(ctx, utxo.TransactionID, utxo.OutputIndex, m.GetOptions(false)...)
```

Comment:
The m.GetOptions is running in the loop. Consider building an opts outside the loop